### PR TITLE
#242 Fix for alarm playing twice

### PR DESCRIPTION
**Pull request by rawlin**
## Short Description
The notification triggering the fullScreenIntent also had the same alarm sound causing the issue
## Issue
https://github.com/SimpleMobileTools/Simple-Clock/issues/242

### CODE REVIEW

**Comment by rawlin** [comment]
> Alarm sound can be customized though, what about that case? I guess this just plays the default one.

The alarm sound is also played in `RemiderActivity` through MediaPlayer which is started by this notification. So there is no need to set a sound for the notification here

**Comment by tibbi** [comment]
it always plays the default audio on lockscreen to me, plus it also vibrates, even through the vibrations are disabled at the Edit alarm dialog. It happens on my Samsung with Android 13, try reproducing it with some other device.

**Comment by rawlin** [comment]
Got the issue! A notification channel once created stays forever, hence the `if` block in which the audio properties are now removed never executes if the channel already exists.

I've refactored the code and done some cleanup of the previous notification channel.

It should work fine now, irrespective if it is a fresh install or not

File: app/src/main/kotlin/com/simplemobiletools/clock/receivers/AlarmReceiver.kt [hunk 1]
```diff
@@ -31,8 +34,9 @@ class AlarmReceiver : BroadcastReceiver() {
             if (isOreoPlus()) {
 
                 val notificationManager = context.getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
-                if (notificationManager.getNotificationChannel("Alarm") == null) {
-                    NotificationChannel("Alarm", "Alarm", NotificationManager.IMPORTANCE_HIGH).apply {
+                if (notificationManager.getNotificationChannel(ALARM_NOTIFICATION_CHANNEL_ID) == null) {
+                    oldNotificationChannelCleanup(notificationManager) // cleans up previous notification channel that had sound properties
```
Comment (tibbi) [inline app/src/main/kotlin/com/simplemobiletools/clock/receivers/AlarmReceiver.kt:38]: have you tested this with multiple alarms with different audio being set up too?

Comment (rawlin) [reply app/src/main/kotlin/com/simplemobiletools/clock/receivers/AlarmReceiver.kt:38]: Yes I have

**Comment by tibbi** [comment]
vibrations still misbehave when the screen is on, but there is another task for that, so Ill accept this. Thanks

**Pull request merged**